### ISSUE TITLE
Add option to import unencrypted JSON account

### ISSUE THREAD

**Issue by 3esmit** [paritytech/parity#4286]
Currently parity don't support this, and can be confusing to some users. The only way to do it is opening the JSON file and using import raw private key.

**Comment by gavofyork** [comment]
where is the format specified?

**Comment by 3esmit** [comment]
Create an wallet in MEW, save the unencrypted version, try to import it on parity.

**Comment by rphmeier** [comment]
@3esmit Unless there's a specification there's a pretty high chance that the format might change over time and break any implementation we might make of it.

**Comment by 3esmit** [comment]
I opened this issue due an user having only saved their MEW unencrypted json file, and was having trouble in importing it to Parity. I reproduced that and confirmed that the only way to do it was opening the JSON file and extracting manually the raw private key and using Parity raw private key import.

See this example of Mnemonic in MEW:
![image](https://cloud.githubusercontent.com/assets/224810/22841337/5be9ce42-efb8-11e6-88de-75b7b9345311.png)

It could be used something like this to able multiple types of private key formats, showing account balances of possible accounts.
